index.js

```diff
@@ -530,6 +530,59 @@ module.exports = {
 				)
 				.catch(err => console.error(err));
 		},
+		dailyRateChange({ synths = [], max = 100 }) {
+			const IGNORE_SYNTHS = ['XDR', 'XDRB', 'nUSD', 'sUSD'];
```

Comment:
What's `XDRB`?

Reply:
no idea, but it is in the list of synths, so I am just excluding it since I have never heard of it